packages/peregrine/src/index.js

```diff
@@ -21,6 +21,7 @@ export { default as List, Items, Item } from './List';
 export { default as Page } from './Page';
 export { default as Price } from './Price';
 export { default as Router } from './Router';
+export { default as Peregrine } from './Peregrine';
```

Comment:
Yea so I'd want to alias this as `PeregrineContextProvider` and in venia `import { PeregrineContextProvider as Peregrine }`.

Reply:
Made these name and alias changes in bf94de4